### PR TITLE
track transaction attributes

### PR DESCRIPTION
**Pull request by eli-d**

### CODE REVIEW

**Review by af-afk — APPROVED** [review]
Needs some integration tests

**Review by glo12345 — APPROVED** [review]

File: tests/pipeline/docker-compose.yml [hunk 1]
```diff
@@ -196,3 +204,24 @@ services:
...
+    build:
+      context: .
+      dockerfile: Dockerfile.tests
+    depends_on:
+      - rabbit
+      - redis
+      - postgres
+      - timescale
+      - spooler
+      - microservice-ethereum-track-winners-util-mining
```
Comment (iscke) [inline tests/pipeline/docker-compose.yml]: nit: if this service is used for multiple things we should probably remove the "util-mining"

Comment (iscke) [reply tests/pipeline/docker-compose.yml]: though if you're using the envs maybe that's fine.

possibly rename the services / envs to make it clear they're not util mining specific anymore

File: tests/pipeline/transaction_attributes_test.go [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+		// ignored
+		ToWinAmount:  payouts,
+		TokenDetails: tokenDetails,
+		Application:  0,
+	}}
+
+	queue.SendMessage(spoolerInputQueue, announcement)
+
+	// payout, block numbers
+	logData_ := "0x0000000000000000000000000000000000000000000000000000000000020f9c000000000000000000000000000000000000000000000000000000000425ebd5000000000000000000000000000000000000000000000000000000000425ebf9"
```
Comment (iscke) [inline tests/pipeline/transaction_attributes_test.go]: not sure how hard it is to do but generating this programmatically would be nice

**Pull request merged**